app/src/main/java/com/hedvig/app/feature/embark/passages/previousinsurer/PreviousInsurerFragment.kt

```diff
@@ -45,7 +61,11 @@ class PreviousInsurerFragment : Fragment(R.layout.previous_insurer_fragment) {
                 onShowInsurers()
             }
             continueButton.setHapticClickListener {
-                onContinue()
+                if (featureManager.isFeatureEnabled(Feature.INSURELY_EMBARK)) {
```

Comment:
I think the insurely-functionality needs to be split out of this fragment - this fragment powers `EmbarkPreviousInsuranceProviderAction`, while insurely should be shown upon `EmbarkExternalInsuranceProviderAction`. 😬

Reply:
Aha! So previous insurance and external insurance are different use cases? What are the difference?

Reply:
They are indeed different! The former merely allows the selection of your previous insurer (no additional interactions), while the latter combines selection of your previous insurer with the possibility of fetching some details about that former insurer with Insurely.

Reply:
I see. Can we look at `InsuranceProvider.hasExternalCapabilities` and only start the insurely flow if its set? And then use the same embark fragment (`EmbarkPreviousInsuranceProviderAction`)

Reply:
I think we need to look at whether the action in the passage is `EmbarkPreviousInsuranceProviderAction` or `EmbarkExternalInsuranceProviderAction` and use that as the basis for whether we should show the insurely UI, but we can probably re-use the same fragment for the implementation 🙂

Reply:
Ok cool, I'll add this in another PR if that makes sense?

Reply:
I guess we could technically do that, as long as we remember to fix it before we toggle the feature flag on 😄